js/templates/modals/listingDetail/listing.html

```diff
@@ -2,11 +2,11 @@
 
 <div class="topControls gutterHSm flex">
   <% if (ob.vendor) { %>
-  <div class="contentBox clrP clrBr clrT">
-    <div class="padSm gutterHSm overflowAuto flexVCent">
+  <div class="contentBox clrP clrSh3 clrBr clrT">
+    <div class="padSm gutterHSm overflowAuto margRSm flexVCent">
```

Comment:
Something funny is going on, I just re-tested it, on my computer it looks like your first screenshot regardless of whether that class is there.